added countryname (#21)
ipinfo/rust: README.md

* Structured and type checked query results.
* Bulk IP address lookup using IPinfo [batch API](https://ipinfo.io/developers/batch).

#### Internationalization

> When looking up an IP address, the `response` includes a `country_name` which includes the country name based on American English and `is_eu` which returns `true` if the country is a member of the European Union (EU). 

Reviewer: ```suggestion
When looking up an IP address, the `response` includes `country_name` which is the country name based on American English and `is_eu` which returns `true` if the country is a member of the European Union (EU). 
```

> ```rust 

Reviewer: ```suggestion

```rust 
```

println!("{}: {}", "8.8.8.8", r["8.8.8.8"].country_name) // United States
println!("{}: {}", "8.8.8.8", r["8.8.8.8"].is_eu) // false
```

> It is possible to return the country name in other languages and change the EU countries by setting the paths of `country_file_path` and `eu_file_path` when creating the `IPinfo` client.

Reviewer: ```suggestion

It is possible to return the country name in other languages and change the EU countries by setting the paths of `country_file_path` and `eu_file_path` when creating the `IPinfo` client.
```

> The file must be a `.json` file with the following structure:
> [countiries.json](./src/countries.json)
> [eu.json](./src/eu.json)

Reviewer: ```suggestion
The files must be `.json` files with structures matching the following:

- [countiries.json](./src/countries.json)
- [eu.json](./src/eu.json)
```

```rust

> let config = IpInfoConfig { country_file_path: Some("custom.json".to_string()), eu_file_path: Some("custom_eu.json".to_string()), ..Default::default() };

Reviewer: ```suggestion
let config = IpInfoConfig {
    country_file_path: Some("custom.json".to_string()),
    eu_file_path: Some("custom_eu.json".to_string()),
    ..Default::default()
};
```
